src/databases/Cale/Cale.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0"?>
   <Plugin name="Cale" type="database" label="Visit Plugin to read Cale PDB dumps" version="1.1" enabled="true" mdspecificcode="false" engspecificcode="false" onlyengine="false" noengine="false" dbtype="STSD" haswriter="false" hasoptions="false" filePatternsStrict="false" opensWholeDirectory="false">
     <LIBS>
-      silo 
+      pdb
```

Comment:
why change from silo to pdb?

Reply:
pdb is just an alias for silo. I had a talk with @markcmiller86 and he prefers we keep the pdb alias, so I have that fix in another branch (along with other minor fixes) to be PR'd after this merge.

Reply:
gotcha, makes sense.

Reply:
I need to modify my comment. I had previously changed 'pdb' to 'silo' with #20328 when I created 'visit_import_third_party'.  Then I had the talk with Mark, and reverted those changes with this PR, and created the 'pdb' library as an alias to 'silo' in FindSilo.cmake.  So there is no future PR that will be doing any changes to 'pdb' and 'silo'.

Reply:
Does that mean this change should be reverted?

Reply:
No. It was originally `${PDB_LIB}` . I changed it to `silo` in PR #20328.  `silo` is the new imported target form of the silo library.  Since  `${PDB_LIB}` was set to `${SILO_LIB}`, I thought it safe to do away with the pdb-is-the-same-as-silo extra vars.   Since Mark wants to keep the pdb alias (in case we ever in the future build PDB proper instead of PDB lite inside Silo), I  created `pdb` as a library alias for `silo`, and changed back the plugins that utilize PDB to link with `pdb` instead of `silo`.